### PR TITLE
Breadcrumb for Systematik

### LINKED ISSUE #353: Implement breadcrumb for Systematik
Mail H.F., 2016-10-26

> Wenn wir es unseren Nutzenden generell erleichtern wollen, nach vielem Sichten, Blättern oder bei Unterbrechung die Suchanfrage wieder zu erinnern, dann hielte ich es zudem für konsequent, den Suchpfad insgesamt wiederzugeben. Auch sehr bekannte Anwendungen wie Amazon arbeiten so, wenn Sie dort etwa nach Büchern, dann nach Romanen (ggf. weiter untergliedert) suchen.

> "Narrensicher" wäre es dann, wenn wir also den Pfad bis zur untersten Systemstelle genau abbildeten. Im Grunde wäre das auch des Guten nicht zu viel, denn bei einer Reihe unserer Systemstellen fragen sich Nutzende sicher, in welchem Fachgebiet sie nun fündig geworden waren. Natürlich könnte ohne den vollständigen Pfad eine Sucheingabe z.B. nach "Private Träger" erfolgen. Doch wäre es wohl einfacher, die Pfadangabe würde auch hier die übergeordneten Einstiege mit anbieten. 

> Weil das nun allerdings über mehrere Stufen führen und recht lang werden kann, ließe sich hier einmal der Vorteil unserer Klassifikation nutzen. Da unsere Notationen rein numerisch sind, bieten sie auf der jeweiligen Klassifikationsstufe problemlos und sofort einsichtig das Hauptfachgebiet und führen ebenso zur gewünschten Ebene. Anstelle des gesamten Pfades brauchten wir dann also nur die Notation zur Klassenbezeichnung hinzuzusetzen, was dann lediglich sechs weitere Zeichen erforderte. Im Beispiel hier könnte das dann so aussehen:

> 626 Treffer in: Sachgebiete / Sachsystematik / 522030 Private Träger

### PR DESCRIPTION
**Pull request by fsteeg**
Will resolve #353

Deployed to staging:
http://test.nwbib.de/search?nwbibsubject=http%3A%2F%2Fpurl.org%2Flobid%2Fnwbib%23s522030

### CODE REVIEW

**Comment by acka47** [comment]
Looks good as asked for by stakeholders. When you chose another filter on top (e.g. "online resources") the information disappears though, see e.g. http://test.nwbib.de/search?medium=http%3A%2F%2Frdvocab.info%2FtermList%2FRDACarrierType%2F1018&nwbibsubject=http%3A%2F%2Fpurl.org%2Flobid%2Fnwbib%23s142080. 

+1 (Although I think this is an isolated fix that only works for this special problem and that we should rather work on a consistent breadcrumb functionality.)

**Comment by fsteeg** [comment]
> +1 (Although I think this is an isolated fix that only works for this special problem and that we should rather work on a consistent breadcrumb functionality.)

That we actually should work on? Or that we should build if further requirements around this come up?

**Comment by acka47** [comment]
> That we actually should work on? Or that we should build if further requirements around this come up?

Considering our resources, the second.

File: app/views/search.scala.html [hunk 1]
```diff
@@ -50,11 +51,17 @@
 	if(raw.contains("multiVolumeWork")) {select(("Band","Bände"))} else if (raw.contains("series")) {select(("Serienband","Serienbände"))} else if (raw.contains("containedIn")) {select(("Beitrag","Beiträge"))} else {""}
 }
 
-@nwbibLabel() = @{
-	if(!nwbibsubject.isEmpty && !nwbibsubject.contains(",") && Seq(nwbibspatial, person, name, subject, id, publisher, issued, medium, owner, t, set, location, word, corporation, raw).forall(_.isEmpty)){
-		s" in Sachsystematik zu '${Lobid.facetLabel(Seq(nwbibsubject),"","")}'"
```
Comment (dr0i) [inline app/views/search.scala.html]: I know this is a very general question, but:
those long lines are not very convenient to read. What speaks against linebreaks after at least 100 chars?

**Comment by dr0i** [comment]
+1

File: app/views/search.scala.html (hunk 1 above)
Comment (fsteeg) [inline app/views/search.scala.html]: Don't know if true here, but often it's due to either some formatting requirement in the template (line breaks becoming line breaks in the resulting HTML), or a requirement of the embedded scripts, which can get invalid syntax with some line breaks. I don't like these long lines either, so feel free to try adding line breaks locally when you encounter things like this in reviews, would be great to get rid of some of these.

**Pull request merged**